### PR TITLE
User1story

### PR DESCRIPTION
**Pull request by sherimichael**

### CODE REVIEW

File: rss-client/src/app/LandingPage/landing-page.module.ts [hunk 1]
```diff
@@ -33,4 +33,3 @@ import { reducer } from 'src/app/LandingPage/Auth/reducers/user.reducer';
   ]
 })
 export class LandingPageModule { }
-
```
Comment (GattieTime) [inline rss-client/src/app/LandingPage/landing-page.module.ts:36]: Please do not just delete empty lines.

File: rss-client/src/app/Quiz/employee/invalid-quiz/invalid-quiz.component.ts [hunk 2]
```diff
@@ -15,11 +15,20 @@ export class InvalidQuizComponent implements OnInit {
...
-  
+  // openDialog() {
+  //   const dialogRef = this.dialog.open(InvalidQuizDialogContent);
+  //   this.open = true
+  //   dialogRef.close().subscribe(() => {
+  //     this.open = false
+  //   });
+  // }
 
 }
```
Comment (GattieTime) [inline rss-client/src/app/Quiz/employee/invalid-quiz/invalid-quiz.component.ts:34]: Why is commented code being added?

File: rss-client/src/app/Quiz/employee/post-test/post-test.component.ts [hunk 3]
```diff
@@ -16,6 +18,17 @@ export class PostTestComponent implements OnInit {
...
+
+  //quiz: Quiz;
+
+  //test this lol
+  // attempt() {
+  //   this.quizservice.subtractAttempt(this.quiz.quizAttempt);
+  // }
+  
+
 }
```
Comment (GattieTime) [inline rss-client/src/app/Quiz/employee/post-test/post-test.component.ts:34]: Why adding commented code?

File: rss-client/src/app/Quiz/service/quiz.service.ts [hunk 4]
```diff
@@ -157,12 +166,47 @@ export class QuizService {
...
+  //sends request to AnswersBankController - create this method based on userscoreid
+  getAnswersByAttemptId(id) {
+    return this.httpclient.get<any[]>(this.url+'/answer/'+id);
+  }
+
+  //userId: User;
+  // accId: Account;
+ // quizId: Quiz;
+  // userScoreId: number;
+
```
Comment (GattieTime) [inline rss-client/src/app/Quiz/service/quiz.service.ts:195]: Reason for these comments?

File: rss-client/src/app/User/discount/discount.component.ts [hunk 5]
```diff
@@ -12,7 +12,6 @@ import { UserService } from '../services/user.service';
   selector: 'app-discount',
   templateUrl: './discount.component.html',
   styleUrls: ['./discount.component.scss']
-
 })
```
Comment (GattieTime) [inline rss-client/src/app/User/discount/discount.component.ts:15]: empty line

File: rss-client/src/index.html [hunk 6]
```diff
@@ -1,12 +1,12 @@
...
-      <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500&display=swap" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+<head>
+  <meta charset="utf-8" />
+  <title>RSS</title>
+  <base href="/" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet"> 
 </head>
```
Comment (GattieTime) [inline rss-client/src/index.html:10]: Please do not only change indentations.

**Review by GattieTime — CHANGES_REQUESTED** [review]
Please review comments and respond or revert where necessary.

File: rss-client/src/app/Quiz/employee/invalid-quiz/invalid-quiz.component.ts (hunk 2 above)
Comment (sherimichael) [reply rss-client/src/app/Quiz/employee/invalid-quiz/invalid-quiz.component.ts:34]: This commented out code will be removed.

File: rss-client/src/app/Quiz/service/quiz.service.ts (hunk 4 above)
Comment (sherimichael) [reply rss-client/src/app/Quiz/service/quiz.service.ts:195]: These should not be included.

File: rss-client/src/app/Quiz/employee/post-test/post-test.component.ts (hunk 3 above)
Comment (sherimichael) [reply rss-client/src/app/Quiz/employee/post-test/post-test.component.ts:34]: This code should be removed.

File: rss-client/src/app/User/discount/discount.component.ts (hunk 5 above)
Comment (sherimichael) [reply rss-client/src/app/User/discount/discount.component.ts:15]: Remove empty line.

File: rss-client/src/index.html (hunk 6 above)
Comment (sherimichael) [reply rss-client/src/index.html:10]: We are not sure how the tabbing happened. This should match the dev branch indentations.

File: rss-client/src/app/Quiz/employee/post-test/post-test.component.ts (hunk 3 above)
Comment (sherimichael) [reply rss-client/src/app/Quiz/employee/post-test/post-test.component.ts:34]: removed

File: rss-client/src/index.html (hunk 6 above)
Comment (sherimichael) [reply rss-client/src/index.html:10]: resolved

File: rss-client/src/app/LandingPage/landing-page.module.ts (hunk 1 above)
Comment (sherimichael) [reply rss-client/src/app/LandingPage/landing-page.module.ts:36]: resolved

**Review by GattieTime — CHANGES_REQUESTED** [review]
Some requested changes were not resolved.

**Review by GattieTime — APPROVED** [review]

**Pull request merged**